demo/c/test/test_orca_c.py

```diff
@@ -0,0 +1,76 @@
+#
+#    Copyright 2024 Picovoice Inc.
+#
+#    You may not use this file except in compliance with the license. A copy of the license is located in the "LICENSE"
+#    file accompanying this source.
+#
+#    Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+#    an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+#    specific language governing permissions and limitations under the License.
+#
+
+import os.path
+import subprocess
+import sys
+import unittest
+
+from test_util import get_model_paths
+
+TEXT = "It doesn't matter how slowly you go, as long as you do not stop!"
```

Comment:
It doesn't need to happen now, but I think it would be much better and easier if we remove the test data and place it in a [JSON file](https://github.com/Picovoice/rhino/blob/master/resources/.test/test_data.json). This way, we won't have to include this string and its corresponding WER in each SDK.

Reply:
Moved the test data to the resource folder